portfolio.css

```diff
@@ -0,0 +1,96 @@
+* {
+	margin: 0;
+	font-size: 20px;
+	color: #ff4716;
+	overflow: auto;
+}
+
+.navbar a {
+	padding: 50% 0 0 5%;
+	text-decoration: none;
+	margin: 50% 0 50% 0;
+	position: relative;
+	top: 30px;
+	font-size: 1.5rem;	
+}
+
+.navbar {
+	background-color: #feeeeb;
```

Comment:
love this colour, it's all about using hex to make funny words rather than nice looking colours ^_^

Reply:
Now I've got to see if I can make any swear words out of hex for when my code goes wrong